build_artifacts/v2/v2.2/v2.2.0/gpu.env.in

```diff
@@ -11,6 +11,7 @@ conda-forge::amazon_sagemaker_sql_editor[version='>=0.1.12,<1.0.0']
 conda-forge::amazon-sagemaker-sql-magic[version='>=0.1.3,<1.0.0']
 conda-forge::langchain[version='>=0.2.16,<1.0.0']
 conda-forge::fastapi[version='>=0.115.2,<1.0.0']
+conda-forge::tornado[version='>=6.3.3,<7.0.0']
```

Comment:
I dont think we are supposed to add dependencies in PR. We should create a separate PR for adding new dependencies and SMD team will add it in next release.

Reply:
ack, will remove